src/Microsoft.DotNet.Build.Tasks/MSBuildDependencyResolver.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using NuGet.DependencyResolver;
+using NuGet.Frameworks;
+using NuGet.LibraryModel;
+using NuGet.ProjectModel;
+using NuGet.Versioning;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.NuGet.Build.Tasks
```

Comment:
Yes, it's coming from Jason's work that's going on in VS.  If we care we can rename to our own namespace and then do the same every
time we take a drop of the code, but I'd prefer to just leave as-is.